hoot-core/src/main/cpp/hoot/core/conflate/splitter/CornerSplitter.cpp

```diff
@@ -110,6 +112,7 @@ void CornerSplitter::splitCorners()
         {
           LOG_DEBUG("splitting way with delta: " << delta);
           _splitWay(pWay->getId(), nodeIdx, pWay->getNodeId(nodeIdx));
+          split = true;
```

Comment:
I like this Boolean flag better than the break.

Reply:
Me too. It's less like a `goto` ...